packages/overlay/src/composables/position.ts

```diff
@@ -64,16 +59,6 @@ export function usePosition(panelEl: Ref<HTMLElement | undefined>) {
   }
 
   onMounted(() => {
-    setTimeout(() => {
-      setWindowSize()
-    }, 200)
-
-    bringUp()
```

Comment:
Why do we remove the `bringUp` call in here?

Reply:
My mistake.